tools/mo/openvino/tools/mo/middle/ConvertGroupedStridedSlice.py

```diff
@@ -91,6 +91,10 @@ def find_and_replace_pattern(self, graph: Graph):
 
             sorted_out_nodes = sorted(out_nodes, key=lambda n: list(n.slices))
             out_nodes = unique_by(sorted_out_nodes, strided_slices_equality)
+            # if there is only one StridedSlice out_node with unique 'slices',
+            # there is nothing to optimize, continue to the next data node
+            if len(out_nodes) <= 1:
+                continue
```

Comment:
It is correct that you recover the previous behavior but in case several unique SS operation they need to be folded into the single SS operation. And it seems to be done on a line 136 but for some reason something goes wrong with it.